core/lib/generators/refinery/form/templates/app/models/refinery/namespace/setting.rb.erb

```diff
@@ -1,3 +1,5 @@
+require 'zilch/authorisation/users_manager'
```

Comment:
This isn't required if this is required in the `Refinery::Core::BaseModel` already.